learn/configuration/typo_tolerance.md

```diff
@@ -16,15 +16,22 @@ Typo tolerance is enabled by default, but you can disable it if needed:
 
 <CodeSamples id="typo_tolerance_guide_1" />
 
-Disabling typo tolerance will mean Meilisearch no longer applies typo tolerance to match queries at search time, but it will still [sort results by increasing number of typos](#impact-of-typo-tolerance-on-the-typo-ranking-rule).
+Disabling typo tolerance will mean Meilisearch no longer applies typo tolerance to match queries at search time. However, it will still [sort results by increasing number of typos](#impact-of-typo-tolerance-on-the-typo-ranking-rule).
 
-### `disableOnAttributes`
+### `minWordSizeForTypos`
 
-You can disable the typo tolerance feature for a specific attribute by adding it to `disableOnAttributes`. The code sample below disables typo tolerance for `title`:
+By default, Meilisearch applies typo tolerance to a query if its length is at least 5 characters. To accept two typos, the query should be at least 9 characters.
```

Comment:
```suggestion
By default, Meilisearch applies typo tolerance to a query term if its length is at least 5 characters. To accept two typos, a query term should be at least 9 characters.
```

By reading this sentence one could understand that the type tolerance feature will be triggered according to the total size of the `q` field which is not true. It is the individual words in `q` on which the typo tolerance feature will be triggered.